pkg/build/starnix.go

```diff
@@ -86,6 +94,7 @@ func (st starnix) build(params Params) (ImageDetails, error) {
 		30*time.Second,
 		params.KernelDir,
 		ffxBinary,
+		"--no-environment",
```

Comment:
The `ffx product` command doesn't rely on ffx's default build config, so it's intentionally not included here.